Gemfile.lock

```diff
@@ -723,4 +727,4 @@ RUBY VERSION
    ruby 3.1.4p223
 
 BUNDLED WITH
-   2.5.6
+   2.3.26
```

Comment:
:(

Reply:
Don't worry, this should get fixed when I resolve merged conflicts on the Rails update PR